webrtc.html

```diff
@@ -399,23 +399,31 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>Let <var>p</var> be the result of transforming
```

Comment:
It's from http://www.w3.org/2001/tag/doc/promises-guide#shorthand-reacting
See also other comments in https://github.com/w3c/webrtc-pc/pull/165